structuredheadings/plugin.js

```diff
@@ -369,8 +364,10 @@
             var block = elementPath.block;
 
             if (block && isNumbered(editor, block)) {
-              this.setValue(editor.config.autonumberCurrentStyle,
-                        editor.config.autonumberCurrentStyle);
+              this.setValue(
+                self.currentPreset,
+                self.currentPreset
```

Comment:
it didn't make sense to store this on the editor config, and was super annoying to find afterwards